src/PrestaShopBundle/Entity/Repository/CmsPageRepository.php

```diff
@@ -26,7 +26,7 @@
 
 namespace PrestaShopBundle\Entity\Repository;
```

Comment:
This is wrong namespace for object model.
If you want using object model, you must introduce the ObjectModel repository under adapter.
This namespace however is supposed to be used by doctrine entities only.
That is also why phpstan is complaining about using legacy CMS object model inside bundle namespace.